File: swarm_input/word_count.py
def word_count(text):
    counts = {}
    for w in text.lower().split():
        counts[w] = counts.get(w, 0) + 1
    return counts

text = "the quick brown fox jumps over the lazy dog the fox"
for word, count in sorted(word_count(text).items(), key=lambda x: x[1], reverse=True)[:5]:
    print(f"{word}: {count}")

File: swarm_input/word_count.rb
def word_count(text)
  counts = Hash.new(0)
  text.downcase.split.each { |w| counts[w] += 1 }
  counts
end

text = "the quick brown fox jumps over the lazy dog the fox"
word_count(text).sort_by { |_, v| -v }.first(5).each do |word, count|
  puts "#{word}: #{count}"
end
